RFC-by-stage/1-approved/remove-pancake.md

```diff
@@ -0,0 +1,23 @@
+---
+Status: `Proposal` # Please do not change this.
+Implementer: # It will be changed upon merging and as it moves through the RFC stages
+---
+
+# Remove pancake
+
+## The issue to be solved
+
+Pancake has been built (by me) a while ago and solved a couple very specific problems outlined in [this article](https://medium.com/dailyjs/npm-and-the-front-end-950c79fc22ce).
+Since 2018 the front end space has evolved significantly and shipping CSS files via npm isn't an issue anymore.
+In-fact shipping CSS files that are automatically imported by the prod bundle located inside a `node_modules/` folder causes significant issues to loaders like webpack which now require the CSS loader to be specified and the specifically allow the `node_modules/` path.
+
+## A short description of the solution
+
+We should remove pancake from the `postinstall` npm hook and deliver the CSS separately in the npm payload.
```

Comment:
I read this that we will also still export Sass.

Reply:
No Sass will be exported I don't think. That's too hard as you need the dependency tree also installed and include it in the right order

Reply:
OK. Can you confirm that you are saying this would remove Sass? if so, perhaps worth capturing.